commit 9816df87d6a46999f2892c22227d5c0b1f21662b (refs/changes/96/3896/3)
Author: Mike Zeller <mike@mikezeller.net>
Date:   2018-05-02T10:31:03-07:00 (1 year, 5 months ago)
    
    OS-6922 test-vrrp-nics should use valid mac addrs
    Reviewed by: Dave Eddy <dave.eddy@joyent.com>
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/src/vm/tests/test-vrrp-nics.js b/src/vm/tests/test-vrrp-nics.js
index 14ef8aeb..890fbc9c 100644
--- a/src/vm/tests/test-vrrp-nics.js
+++ b/src/vm/tests/test-vrrp-nics.js
@@ -1,4 +1,4 @@
-// Copyright 2015 Joyent, Inc.  All rights reserved.
+// Copyright 2018 Joyent, Inc.  All rights reserved.
 //
 // Test invalid nic tag detection
 //
@@ -247,7 +247,7 @@ test('vrrp vnics: updating', function (t) {
                 payload: {
                     add_nics: [
                         {
-                            mac: '52:31:98:52:5d:07:d0',
+                            mac: '52:31:98:52:5d:07',
                             nic_tag: 'external',
                             ip: 'dhcp',
                             vrrp_vrid: 4
@@ -554,7 +554,7 @@ test('vrrp vnics: updating', function (t) {
 test('create with both mac and vrrp_vid', function (t) {
     expectCreateError({
         nics: [ {
-                mac: '52:31:98:52:5d:07:d1',
+                mac: '52:31:98:52:5d:07',
                 vrrp_vrid: 9
         } ],
         t: t,
